src/main/java/lotto/model/LottoNumberGenerator.java

```diff
@@ -0,0 +1,24 @@
+package lotto.model;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumberGenerator {
+
+    private static List<Integer> lottoNumbers = IntStream.range(1, 45)
```

Comment:
아 제가 지난번에 잘못 피드백 드렸습니다 ㅜ range가 아니고 rangeClosed를 써야하네요. 죄송합니다
```suggestion
    private static List<Integer> lottoNumbers = IntStream.rangeClosed(1, 45)
```